src/sentry/templates/sentry/organization-login.html

```diff
@@ -15,6 +15,30 @@
     input: '#id_registration_password',
     element: '.password-strength',
   });
+  // HACK(jferge): inline getCookie function from our utils,
+  // update the csrf token value in a rudimentary way (timer based)
+  // on the SSO login form
+  // to help out when users have multiple sentry tabs and are logged out
+  function getCookie(name) {
+    if (document.cookie && document.cookie !== '') {
+      const cookies = document.cookie.split(';');
+      for (let i = 0; i < cookies.length; i++) {
+        const cookie = cookies[i].trim();
+        // Does this cookie string begin with the name we want?
+        if (cookie.substring(0, name.length + 1) === name + '=') {
+          return decodeURIComponent(cookie.substring(name.length + 1));
+        }
+      }
+    }
+    return null;
+  }
+  setInterval(function() {
```

Comment:
huh....

Uhhh.... do we have some sort of timeout killswitch here? we're just gonna be running this every 200ms forever unless they complete login?

I'm not actually sure on best practices for something like this... 🤔

Reply:
Just to confirm, navigating away from this page will be served a separate source file right?
It's not like we're doing some single page stuff with this template? (potential for zombie runners or something)

Reply:
no killswitch, forever ♾️  yes. don't think this is a problem

yes, navigating away will be served from a separate source file. don't think zombie running would be a problem.

Reply:
If you don't want to use a listener on `localStorage`, can we do something like:

```
clearInterval(window.loginLoopCheck)
window.loginLoopCheck = setInterval(...)
```

so we'll only have 1 loop.

Reply:
@leedongwei how would more than 1 loop get created in the first place? This view isn't inherited from anywhere, etc.

Reply:
No idea but we're running JS in someone else's browser with unknown extensions that can modify the runtime and this is an iffy pattern to begin with. 

It's 1 extra line of code to be a bit more defensive. Up to you 🤷‍♂️, LGTM otherwise,

Reply:
We've done something similar at Specto and used the localStorage listener. Iirc we would refresh the session token on all tabs once user logged in on one tab or logout the user on all tabs once they logged out on one tab. LocalStorage listener worked well and there's no need to poll infinitely, just remember to not actually store the token because localStorage is shared and can easily be enumerated